src/components/MaxFeeAndSubmitToken/MaxFeeAndSubmitToken.less

```diff
@@ -0,0 +1,27 @@
+.warning-fee-low {
```

Comment:
Why did you create a copy of the MaxFreeAndSubmit.less component? Do you have any differences?

Reply:
to separate the button "Send" and give it different styles